config/config.go

```diff
@@ -212,6 +212,10 @@ type ServerConfig struct {
 	// once the server is promoted
 	SkipIfReplica bool `ini:"skip_if_replica"`
 
+	// The maximum shared_buffers size in gigabytes that the collector will monitor
+	// pg_buffercache. Defaults to 200 GB.
+	MaxBufferCacheMonitoringGB int `ini:"max_buffer_cache_monitoring_gb"`
```

Comment:
Thoughts on the naming of this setting? It's difficult to find a name that's descriptive without being too long.